### DISCUSSION TITLE
Advanced setting option for remote blockchain IP

### DISCUSSION THREAD

**Discussion by xuedi** [Chia-Network/chia-blockchain-gui#2374] [Ideas]
I have a home server that runs the blockchain & a farmer, but i use the GUI at my desktop, it is very complicated to go into the config files and redirect the GUI to use the blockchain on a different machine. 

In the setting under advanced, it would be nice to just enter the IP address of the remote box, then a yes confirmation for the certs and it should work. Maybe even on first run to ask to run the block chain local or remote ...

**Comment by BrandtH22** [comment]
@xuedi , thank you for the recommendation! Our team will review and work on this as time permits which means it could be some time before it is implemented. 

If you do have coding experience we do also welcome community PRs as that can be a faster implementation of new ideas. 
Let us know if you have any questions and feel free to add any additional details to this ticket as needed :+1:

**Comment by BrandtH22** [comment]
@xuedi  , note I transferred this enhancement request to the GUI repo for better visibility 👍
